main.go

```diff
@@ -140,6 +141,8 @@ func main() {
 
 	gctscript.Setup()
 
+	gctlog.Infof(gctlog.Global, "JSON encoding is set to package `%s`", json.Version)
```

Comment:
I don't believe this is important at an Info level.
I actually don't think we should log it even at verbose + Debug.

Reply:
The intent is just to show that the JSON implementation can be configured. Would it make more sense to log only when a non-default implementation is used and changed to debug?

Reply:
Up to you. Resolve if you're happy with it as is 🙂 
I always try to keep Info limited so that when I look at logs they don't overwhelm with information which isn't relevant.

Reply:
after consideration I am resolving, and add more pain to your log output!